scene/main/viewport.h

```diff
@@ -304,6 +305,8 @@ class Viewport : public Node {
 	ScreenSpaceAA screen_space_aa = SCREEN_SPACE_AA_DISABLED;
 	bool use_debanding = false;
 	float lod_threshold = 1.0;
+	bool use_occlusion_culling = false;
+	int occlusion_rays_per_thread = 512;
```

Comment:
Again would probably make this global and just a good enough default, not much of a point per viewport.